src/manager.cpp

```diff
@@ -392,14 +415,31 @@ static std::map<pid_t, std::string> start_modules(Config& config, MQTTAbstractio
         std::string python_filename = "module.py";
         auto module_path = rs->modules_dir / module_type;
         const auto printable_module_name = config.printable_identifier(module_name);
-        auto binary_path = module_path / binary_filename;
+        auto cpp_binary_path = module_path / binary_filename;
         auto javascript_library_path = module_path / javascript_library_filename;
         auto python_module_path = module_path / python_filename;
+        std::string rust_filename;
+        auto char_count = 0;
+        for (auto& character : binary_filename) {
+            if (char_count > 0 && isupper(character)) {
+                rust_filename += "_";
+                rust_filename += tolower(character);
+            } else {
+                if (isupper(character)) {
+                    rust_filename += tolower(character);
+
+                } else {
+                    rust_filename += character;
+                }
+            }
+            char_count++;
+        }
+        auto rust_binary_path = module_path / "bin" / rust_filename;
```

Comment:
Is this really necessary? I hope not. If so, this should be put into a helper function with a proper name to improve readabilitiy

Reply:
I admit this is the biggest hack here, the easiest would be to just start naming the rust binary targets in CamelCase as well, then we wouldn't need this hacky conversion in the first place. But I'm not sure about the rust conventions here, would this be alright @SirVer ?